jvm-pdf/src/main/groovy/org/asciidoctor/gradle/jvm/pdf/AsciidoctorPdfTask.groovy

```diff
@@ -68,7 +78,12 @@ class AsciidoctorPdfTask extends AbstractAsciidoctorTask {
      *   can be used.
      */
     void setFontsDir(Object f) {
```

Comment:
Setting the single directory should be deprecated.

and then you need two methods: `setFontDirs` and `fontDirs`. Look at https://github.com/asciidoctor/asciidoctor-gradle-plugin/blob/master/jvm/src/main/groovy/org/asciidoctor/gradle/jvm/AsciidoctorJExtension.groovy and see how gemPaths are implemented. You should do the same for fontDirs.

Reply:
There are quite a number of changes required before this can be accepted, but don't despair as this is not a trivial Gradle plugin to contribute to.

Reply:
Never! I will get this done! :D 

I made a new commit. Hope that is better now ?